### ISSUE TITLE
Why is Tanana Zone fire management unit returning 422s?

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/data-api#165]
https://earthmaps.io/taspr/area/FIRE11
https://earthmaps.io/alfresco/flammability/area/FIRE11
https://earthmaps.io/alfresco/veg_type/area/FIRE11

We'd definitely want these to work.

**Comment by charparr** [comment]
Maybe this was an `update` gone wrong?
The following do not work and I'm not spotting a pattern in terms of the polygons themselves:
 - FIRE6
 - FIRE8
 - FIRE10
 - FIRE11

**Comment by charparr** [comment]
Well this is weird. I'm running the main branch locally and some of these queries work on-and-off!
When they work, I see ALFRESCO results as you'd expect, e.g., http://localhost:5000/alfresco/flammability/area/FIRE11
```
{
  "1950-1979": {
    "MODEL-SPINUP": {
      "historical": 0.0039
    }
  }, 
  "1980-2008": {
    "MODEL-SPINUP": {
      "historical": 0.0067
    }
  }, 
  "2010-2039": {
    "5modelAvg": {
      "rcp45": 0.0043, 
      "rcp60": 0.0039, 
      "rcp85": 0.0039
    }, 
```
But I can refresh that same query and get a 500 error

```
aiohttp.client_exceptions.ClientResponseError: 500, message='Internal Server Error', url=URL('http://zeus.snap.uaf.edu:8080/rasdaman//ows?&SERVICE=WCS&VERSION=2.0.1&REQUEST=GetCoverage&COVERAGEID=alfresco_relative_flammability_30yr&SUBSET=X(-128196.43789999932,268219.17540000007)&SUBSET=Y(1426763.6722,2292633.3236999996)&FORMAT=application/netcdf')
```

Could this be a timeout issue?

To replicate, comment out the try/except wrapping (lines 585, 587-588) in alfresco.py.
After that, bounce off these queries a few times.

**Comment by charparr** [comment]
If you try just the request URL (http://zeus.snap.uaf.edu:8080/rasdaman//ows?&SERVICE=WCS&VERSION=2.0.1&REQUEST=GetCoverage&COVERAGEID=alfresco_relative_flammability_30yr&SUBSET=X(-128196.43789999932,268219.17540000007)&SUBSET=Y(1426763.6722,2292633.3236999996)&FORMAT=application/netcdf) the netCDF you'll get back is about 180 MB if you're successful.

If not,
```
<ows:ExceptionReport version="2.0.1" xsi:schemaLocation="http://www.opengis.net/ows/2.0 http://schemas.opengis.net/ows/2.0/owsExceptionReport.xsd">
<ows:Exception exceptionCode="InternalComponentError">
<ows:ExceptionText>NullPointerException</ows:ExceptionText>
</ows:Exception>
</ows:ExceptionReport>
```

**Comment by BobTorgerson** [comment]
After restarting the Rasdaman and Tomcat service on Zeus, things locally worked as expected. This resulted in me digging into the EB instance again, and discovering that for these larger areas, the previous EC2 instance size did not contain enough memory to build the returned data. 

I've increased the production API instance to 16 GBs of RAM now and the previously unavailable fire zones are working as expected. I've updated this document with the new method for creating the single xlarge EB instance: https://docs.google.com/document/d/1Z31-mkDE0skITOuOOMBQwuO2I8jUDuApm7VX-A9v1LA
